tests/ert3/console/integration/test_cli.py

```diff
@@ -28,14 +28,16 @@ def test_cli_no_args():
         ert3.console.main()
 
 
-def test_cli_init(tmpdir):
+@pytest.mark.requires_ert_storage
+def test_cli_init(tmpdir, ert_storage):
     with tmpdir.as_cwd():
         args = ["ert3", "init"]
         with patch.object(sys, "argv", args):
             ert3.console.main()
 
 
-def test_cli_init_twice(tmpdir):
+@pytest.mark.requires_ert_storage
+def test_cli_init_twice(tmpdir, ert_storage):
```

Comment:
Why do we pass `ert_storage` to some of the tests and not all?

Reply:
I am only passing it to the ones that are failing without storage

Reply:
Some use workspace-fixture, which requires ert-storage

Reply:
Aha, I get it! I find it a bit odd that we are then making a fixture that we don't really expect people to interact with, but maybe that is just me :)

Second question: Why do we have tests that are labelled as depending on storage, but that does not need a storage fixture? My immediate response is that it sounds illogical, but I'm probably just missing something...

Reply:
which tests are you refering to?